src/appkit.cs

```diff
@@ -9936,9 +9936,9 @@ partial interface NSImageRep : NSCoding, NSCopying {
 		[Export ("unregisterImageRepClass:")]
 		void UnregisterImageRepClass (Class imageRepClass);
 
-		//[Static]
-		//[Export ("registeredImageRepClasses")]
-		//Class [] RegisteredImageRepClasses ();
```

Comment:
_Fun fact, this has been commented for as long as I could go back in time (git history)._

_Note: before open source and even before files were renamed / moved around._